src/aliceVision/image/io.hpp

```diff
@@ -218,9 +220,18 @@ class ImageWriteOptions
         return *this;
     }
 
+    ImageWriteOptions& storageDataType(EStorageDataType storageDataType)
+    {
+        _storageDataType = storageDataType;
+        _storageDataTypeSet = true;
+        return *this;
+    }
+
 private:
     EImageColorSpace _fromColorSpace{EImageColorSpace::LINEAR};
     EImageColorSpace _toColorSpace{EImageColorSpace::AUTO};
+    EStorageDataType _storageDataType{EStorageDataType::HalfFinite};
```

Comment:
Maybe add an EStorageDataType::Undefined to avoid the extra boolean?

Reply:
Good idea, fixed now.